modules/vlada-m/src/main/java/com/hillel/elementary/javageeks/task6/Replacer.java

```diff
@@ -0,0 +1,41 @@
+package com.hillel.elementary.javageeks.task6;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Replacer {
+    public static String replace(String pathname) throws IOException {
```

Comment:
method should take absolute path to the file, not filename of a classpath resource